### PR TITLE
Add recipe doc for database migrations 

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: doc/recipes/migrating-database-versions.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+## Migrating database versions
+
+Database migrations are done using [Room](https://developer.android.com/training/data-storage/room/migrating-db-versions). These migrations are run 
+when the app is opened for the first time after an update.
```
Comment (Honey14) [inline doc/recipes/migrating-database-versions.md:4]: there's an extra space here, can you format the file, please?

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good overall, left one comment, PTAL

File: doc/recipes/migrating-database-versions.md (hunk 1 above)
Comment (msasikanth) [reply doc/recipes/migrating-database-versions.md:4]: At the end of the line?

Comment (msasikanth) [reply doc/recipes/migrating-database-versions.md]: I pushed the change. PTAL

**Review by Honey14 — APPROVED** [review]

**Pull request merged**